### PR TITLE
Port Ajaxifier to modern JavaScript

### PR DESCRIPTION
**Pull request by marschall**
- Remove dependency on jQuery
- Remove dedicated code in render phase continuation

### CODE REVIEW

**Comment by marschall** [comment]
This PR ports the Ajaxifier to modern JS, this allows us to get rid of the jQuery dependency and the special code in `WARenderPhaseContinuation`

In more detail:
- Replace the special `X-RedirectedTo` header added by `WARenderPhaseContinuation` with `XMLHttpRequest.responseURL`.
- Replace the regex response processing with `responseType = "document"` and `HTMLDocument` API.
- Replace the jQuery ready function `window.addEventListener("DOMContentLoaded"`
- Replace other jQuery functions with `addEventListener`, `closest`, `getAttribute`
- Use `FormData` API to generate the request body
- Replace functions with arrow functions.
- Replace `var` with `let` or `const` where possible
- Assume `window.history.pushState` is present.
- Documentation has been moved to https://github.com/SeasideSt/Seaside/wiki/Ajaxification

I don't actually know what I'm doing, I just copied and pasted stuff from MDN and it seems to work 🤷‍♂️

File: repository/Seaside-Ajaxifier-Core.package/WAAjaxifierLibrary.class/properties.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
-	"commentStamp" : "",
+	"commentStamp" : "pmm 8/25/2022 15:03",
 	"super" : "WAFileLibrary",
-	"category" : "JQuery-Core-Libraries",
+	"category" : "Seaside-Ajaxifier-Core",
```
Comment (marschall) [inline repository/Seaside-Ajaxifier-Core.package/WAAjaxifierLibrary.class/properties.json:4]: Could also be `Javascript-Ajaxifier-Core`, let me know what you think.

Comment (jbrichau) [reply repository/Seaside-Ajaxifier-Core.package/WAAjaxifierLibrary.class/properties.json:4]: That's fine.

**Comment by jbrichau** [comment]
When trying the ajaxified functional tests, I notice that the development toolbar is not properly handled. See screenshot.
This did not occur with the previous version of the ajaxifier, but I have not started investigating why that is.
Screenshot 2022-09-10 at 10 16 22

**Comment by marschall** [comment]
> When trying the ajaxified functional tests, I notice that the development toolbar is not properly handled.

Found it. The `<head>` content was not set properly.

**Review by jbrichau — APPROVED** [review]

**Comment by jbrichau** [comment]
Thanks!

**Pull request merged**